### DISCUSSION TITLE
关于优选IP所在区域的问题

### DISCUSSION THREAD

**Discussion by penghuichou** [vvbbnn00/WARP-Clash-API#177] [Q&A]
用两个不同地区VPS搭建的服务都是优选到美国和荷兰的IP，请问这个区域可以修改么？还是默认就是只能选到这两个区域的IP呢？

**Comment by vvbbnn00** [comment]
目前只有这两个区域的接入点，参照该文件：https://gitlab.com/Misaka-blog/warp-script/-/blob/main/files/warp-yxip/ips-v4.txt?ref_type=heads

**Reply by penghuichou** [reply]
好的，谢谢！
